tensorflow/python/distribute/distribute_lib.py

```diff
@@ -633,17 +638,22 @@ class InputOptions(
       strategy.experimental_distribute_dataset(
           dataset,
           tf.distribute.InputOptions(
-              experimental_prefetch_to_device=False)))
+              experimental_prefetch_to_device=False,
+              replication_mode=InputReplicationMode.PER_WORKER)))
   ```
 
   Attributes:
     experimental_prefetch_to_device: Boolean. Defaults to True. If True, dataset
       elements will be prefetched to accelerator device memory. When False,
       dataset elements are prefetched to host device memory. Must be False when
       using TPUEmbedding API.
+      replication_mode: Replication mode for the input function. Currently, the 
+      InputReplicationMode.PER_WORKER works only under mirrored_strategy for the 
```

Comment:
did you mean to say PER_REPLICA only works for MirroredStrategy?

Reply:
Corrected.